README.md

```diff
@@ -108,6 +108,15 @@ unark(files, new_db, lines = 50000)
 #>  ...Done! (in 0.202945 secs)
 ```
 
+# Output Formats and Custom Read and Write
+
+The `arkdb` package is easily extended to use custom read and write methods allowing you to dictate your own output formats. See `R/streamable_table.R` for examples that include using: 
+
+- Base c/tsv
+- The `vroom` package 
+- Apache arrow's parquet
+- The `readr` package for c/tsv
+
```

Comment:
nice; though I think you wanted to edit the `README.Rmd` and knit that?  

Minor notes: `vroom` is now the backend for `readr` >= 2.0, so not sure if we should still mention it separately?

Reply:
Updated.